### PR TITLE
Replace read-pkg-up lib by the custom js file

### PR DESCRIPTION
**Pull request by 4en1x**
read-pkg-up replacement by custom js file. #232

### CODE REVIEW

File: packages/size-limit/read-pkg-up.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+const path = require('path')
+const { promisify } = require('util')
+const fs = require('fs')
+
+const readFileAsync = promisify(fs.readFile)
```
Comment (ai) [inline packages/size-limit/read-pkg-up.js:5]: Use `fs/promises` instead

File: packages/size-limit/read-pkg-up.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+const readPkg = async cwd => {
```
Comment (ai) [inline packages/size-limit/read-pkg-up.js:7]: Use `let` for code style consistency

File: packages/size-limit/read-pkg-up.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+const path = require('path')
```
Comment (ai) [inline packages/size-limit/read-pkg-up.js:1]: Can you import methods like `let { resolve } = require(…)` to keep consistency with other files

File: packages/size-limit/read-pkg-up.js (hunk 1 above)
Comment (4en1x) [reply packages/size-limit/read-pkg-up.js:5]: I see that configured version of node is `10^ || ...`, and `require('fs/promises')` are supported from version 12, and `require('fs').promises` are supported from version 11

Comment (ai) [reply packages/size-limit/read-pkg-up.js]: Got it 👍

**Review by ai — APPROVED** [review]

File: packages/size-limit/read-pkg-up.js (hunk 2 above)
Comment (4en1x) [reply packages/size-limit/read-pkg-up.js]: Fixed in 84789586cb9d4aa93cb6945325a4d26933dce720

**Comment by ai** [comment]
I will accept it when I will be at home

File: packages/size-limit/read-pkg-up.js (hunk 3 above)
Comment (4en1x) [reply packages/size-limit/read-pkg-up.js]: Fixed in 84789586cb9d4aa93cb6945325a4d26933dce720

**Comment by ai** [comment]
Released in 5.0

**Pull request merged**